package exceptions;

public class Q4 {

	public static void main(String[] args) {

			try {
			    System.out.println("Try block");
			    return;
			} finally {
			    System.out.println("Finally block");
			}


	}

}
